src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", 10);
+    }
+
+    public Figure getRandomFigure() {
+        int randomNumberFirst = random.nextInt(1, 10);
+        int randomNumberSecond = random.nextInt(1, 10);
+        int randomNumberThird = random.nextInt(1,10);
+        int index = new Random().nextInt(RandomFigure.values().length);
```

Comment:
Avoid creating a new instance of Random each time you call `getRandomFigure()`. You should use the existing `random` field instead.